charts/tfy-karpenter/Chart.yaml

```diff
@@ -2,14 +2,14 @@ apiVersion: v2
 name: tfy-karpenter
 description: Truefoundry karpenter provisioner
 type: application
-version: 0.2.0
+version: 0.3.0
```

Comment:
Its a major upgrade let's make it `1.0.0` (optional)

Reply:
1.x.x would imply this is stable when the underlying application itself doesn't provide such a guarantee. Right now we can stick to 0.x.x I think. https://semver.org/#spec-item-4